views/signup.ejs

```diff
@@ -1,24 +1,44 @@
 <!DOCTYPE html>
 <html>
-
 <head>
-
-	<title>Sign Up | CodeManiacs</title>
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link rel="stylesheet" type="text/css" href="/css/sheet.css">
-	<link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
+<title> SignUp | CodeManiacs</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
+<link href="./style.css" rel="stylesheet" type="text/css" media="all" />
+<link href="//fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i" rel="stylesheet">
 </head>
-
 <body>
-	<header>
-		<% include partials/header %>
-    </header>
-    
-    <h1>Sign Up</h1>
-
-	<footer>
-		<% include partials/footer %>
-	</footer>
+	<!-- main -->
+	<div class="main-w3layouts wrapper">
+		<h1><u><b> SignUp CodeManiacs</b></u></h1>
+		<div class="main-agileinfo">
+			<div class="agileits-top">
+				<form action="#" method="post">
+                    <input class="text" type="text" name="Name" placeholder="Full Name" required="">
+					<input class="text" type="text" name="Username" placeholder="Username" required="">
+					<input class="text email" type="email" name="email" placeholder="Email" required="">
+					<input class="text" type="password" name="password" placeholder="Password" required="">
+					<input class="text w3lpass" type="password" name="password" placeholder="Confirm Password" required="">
```

Comment:
Make the attribute `name` to:
Line18: name = "name"
Line19: name = "username"
Line20: name = "email"
Line21: name = "password"
Line22: name = "password2"
all in lowercase